test/framework/src/main/java/org/elasticsearch/index/engine/EngineTestCase.java

```diff
@@ -1444,7 +1444,7 @@ public static void waitForOpsToComplete(InternalEngine engine, long seqNo) throw
         assertBusy(() -> assertThat(engine.getLocalCheckpointTracker().getProcessedCheckpoint(), greaterThanOrEqualTo(seqNo)));
     }
 
-    public static boolean hasSnapshottedCommits(Engine engine) {
+    public static boolean hasAcquiredCommits(Engine engine) {
```

Comment:
Since we're renaming this and I agree, can we rename also `internalEngine.hasSnapshottedCommits();` to hasAcquiredCommits (and in general refactor that code path with renaming "snapshots" to "acquired commits")?